app/controllers/rewards_controller.rb

```diff
@@ -0,0 +1,53 @@
+class RewardsController < ApplicationController
+	before_action :require_login
+
+	def index
+		@family = Member.where(family_id: session[:user_id])
+	end
+
+	def create
+		@reward = Reward.new(reward_params)
+		@reward.family_id = current_user.id
+		render json: @reward
+	end
+
+	def update
+		reward = Reward.find_by(id: params[:id])
+		if reward.update_attributes(reward_params)
+			flash[:success] = "The reward was changed"
+			render json: reward
+		else
+			#flash[:error] = "The reward was not changed. Please try again."
+			#
+		end
+	end
+
+	def destroy
+		reward = Reward.find_by(id: params[:id])
+		if current_user == reward.family
+			if reward.destroy
+				all_rewards = Reward.all
+				render json: all_rewards
+			else
+			end
+		else
```

Comment:
Ideally implement a strategy for dealing with failure, and fill in these else branches. Otherwise just delete them. Having them present but empty just looks bad.